Mathlib/CategoryTheory/Shift/Adjunction.lean

```diff
@@ -198,6 +198,21 @@ class CommShift : Prop where
   commShift_unit : NatTrans.CommShift adj.unit A := by infer_instance
   commShift_counit : NatTrans.CommShift adj.counit A := by infer_instance
 
+lemma commShift_unit [adj.CommShift A] : NatTrans.CommShift adj.unit A := CommShift.commShift_unit
+
+lemma commShift_unit_app [adj.CommShift A] (a : A) (X : C) :
+    (adj.unit.app X)⟦a⟧' = adj.unit.app (X⟦a⟧) ≫ ((F ⋙ G).commShiftIso a).hom.app X := by
+  convert (commShift_unit adj A).comm_app _ a X
+  simp
```

Comment:
The lemma `commShift_unit` is unnecessary as the intended way to use `NatTrans.CommShift` is as follows:
```suggestion
open CommShift in
attribute [instance] commShift_unit commShift_counit

@[reassoc (attr := simp)]
lemma commShift_unit_app [adj.CommShift A] (a : A) (X : C) :
    adj.unit.app (X⟦a⟧) ≫ ((F ⋙ G).commShiftIso a).hom.app X = (adj.unit.app X)⟦a⟧' := by
  simpa using (NatTrans.CommShift.comm_app adj.unit a X).symm
```
(Note that the namespaces/names for the API lemmas of `NatTrans.CommShift` are fixed in #20364, so that it would be reasonable to make this PR a PR which depends on it.)